serialization.md

```diff
@@ -327,7 +327,6 @@ Note: whitespace has been added to this example for clarity. Whitespace is OPTIO
         <ul>
           <li>
             <code>type</code> is usually set to <code>layers</code>.
-	    There is also a Windows-specific value <code>layers+base</code> that allows a base layer to be specified in a field of <code>rootfs</code> called <code>base_layer</code>.
```

Comment:
On Mon, Aug 29, 2016 at 04:11:52PM -0700, Stephen Day wrote:
> @wking Could you please focus your review of the PR on what can
> actually be changed?

If the intention here is to only allow the layers-type rootfses, I
don't see a problem with dropping the field from the spec.  You could
handle the drop/restoration with the same code that translates Docker
↔ OCI media types.  Is there a compatibility issue with that that I'm
missing?

> The presence of these field allows it to internally evolve without
> much effort.

So the idea is to provide space for alternative types that still use
the application/vnd.oci.image.serialization.config.v1+json type?
That's fine with me, but I'd rather you changed the current “usually
set” language to explain how users are supposed to handle configs that
use different values.  MUST they error out if asked to unpack an image
whose rootfs.type they don't understand (I expect so)?  Is there any
way to determine if your image-spec implementation is modern enough to
unpack a particular image (I expect not by comparing version numbers,
but you could probably do this by using your tooling to validate the
image)?